storage/rocksdb/ha_rocksdb.cc

```diff
@@ -16413,6 +16413,19 @@ inline bool ha_rocksdb::is_instant(const Alter_inplace_info *ha_alter_info) {
           static_cast<uint8_t>(Instant_Type::INSTANT_IMPOSSIBLE));
 }
 
+/*
+  DDL trx update both table metadata(DD table) and raw data,
+  - For table metadata(thd_is_dd_update_stmt()==True), always skip bulk loading
+  due to DD will read/restore latest metadata after updating metadata and bulk
+  loading doesn't suppose read before commit
+  - For raw data(thd_is_dd_update_stmt()==False), use bulk-load if requested
+  @return True if DDSE is rocksdb and it is updating table metadata
+*/
+inline bool ha_rocksdb::is_dd_update() {
```

Comment:
```suggestion
inline bool ha_rocksdb::is_dd_update() const {
```
But, `inline` in source file will not be inlined unless LTO is used. Please either remove `inline`, or move to the header file

Reply:
remove inline..

Reply:
Please add `const` too :)